app/scripts/components/common/datepicker/index.scss

```diff
@@ -0,0 +1,52 @@
+@use "uswds-core/src/styles/functions/utilities" as utils;
+
+.react-calendar {
+  z-index: 99999;
+
+  * {
+    font-family: var(--base-font-family);
+  }
+
+  &__navigation {
+    margin-bottom: 0 !important;
+  }
+
+  &__month-view__weekdays__weekday {
+    color: utils.color('gray-90');
+  }
+
+  &__tile--hasActive {
+    background-color: utils.color('primary-vivid') !important;
+    color: utils.color('base-lightest');
+  }
+
+  &__month-view__days__day--weekend {
+    color: utils.color('gray-90') !important;
+  }
+}
+
+.tippy-box {
```

Comment:
(I know this pr is just poc, but for my curiosity) - Would this style be applied globally?

Reply:
Thanks for spotting this, you're right it will be applied globally. Fixed to apply to the scope only.